testdriver/testdriver.go

```diff
@@ -110,13 +133,47 @@ func NewTestDriver(name string, govimPath, testHomePath, testPluginPath string,
 		vimCmd = strings.Fields(e)
 	}
 
+	if c.Debug.Enabled {
+		res.debug = c.Debug
+		vimCmd = append(vimCmd, fmt.Sprintf("-V%d%s", c.Debug.VimLogLevel, c.VimLogPath))
+	}
+
 	res.cmd = exec.Command(vimCmd[0], vimCmd[1:]...)
-	res.cmd.Env = env.Vars
-	res.cmd.Dir = env.WorkDir
+	res.cmd.Env = c.Env.Vars
+	res.cmd.Dir = c.Env.WorkDir
+
+	if res.debug.Enabled {
+		envlist := ""
+		for _, e := range c.Env.Vars {
+			if e != ":=:" {
+				envlist += " " + strings.ReplaceAll(e, " ", `\ `)
+			}
+		}
+
+		fmt.Printf("Test command:\n==========================\npushd %s && %s %s && popd\n==========================\n", c.Env.WorkDir, envlist, strings.Join(res.cmd.Args, " "))
+	}
 
 	return res, nil
 }
 
+func (d *TestDriver) Logf(format string, a ...interface{}) {
+	fmt.Fprintf(d.Log, format+"\n", a...)
```

Comment:
Again, only output if debugging enabled?

Reply:
This should only be going to `d.Log`, which by default only logs to an error file